2017/SEPT/SEPT17/CHEFSUM/CHEFSUM.py

```diff
@@ -1,21 +1,11 @@
-# made by vanguard7
+# Made by Piyush
 def main():
-    t = int(input())
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp5wkxoi0v/2017/SEPT/SEPT17/CHEFSUM/CHEFSUM.py
+++ b/tmp/tmp5wkxoi0v/2017/SEPT/SEPT17/CHEFSUM/CHEFSUM.py
@@ -5,7 +5,7 @@
         l = int(input())
         arr = list(map(int, input().split()))
         print(arr.index(min(arr)) + 1)
-        
+
 
 if __name__ == '__main__':
     main()
```